sides/thai-chile-brussels-sprouts.md

```diff
@@ -0,0 +1,42 @@
+# Thai chile Brussels sprouts
```

Comment:
chile or chili?

Reply:
Great question, [ambiguous answer](https://www.thespruceeats.com/different-spellings-of-chile-2342844). I'm of the persuasion that views 'chile' as the pepper, 'chili' as the stew, and 'Chile' as a chile-shaped country.